src/user.c

```diff
@@ -40,6 +40,16 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
 extern int eos;
 
+/** Lookup an account from a Nick 
+ * @param   nick    pointer to a Nick
+ * @returns an User struct
+ * @note    This is an helper function to ease the migration to an account based 
+ *          paradigm.
+ */
+User *find_account(Nick *nick) {
```

Comment:
Please constify the argument, as the nick pointer will not be modified.

Reply:
Done